node/manifest.ts

```diff
@@ -48,11 +53,23 @@ const generateManifest = ({
   bundles = [],
   declarations = [],
   functions,
+  functionConfig = {},
   importMap,
   layers = [],
 }: GenerateManifestOptions) => {
   const preCacheRoutes: Route[] = []
   const postCacheRoutes: Route[] = []
+  const manifestFunctionConfig: Manifest['function_config'] = Object.fromEntries(
```

Comment:
I think this means that there will be an entry in `function_config` for every function, even if the object is empty. Is that intended? Could we simply omit a function from the object if it doesn't have any configuration properties?

Reply:
Thought the same when I wrote this first. Then I replaced it with https://github.com/netlify/edge-bundler/pull/274/commits/7c497533073b937262212acc4bf423685f1df562, found that to be too much indirection, and went back. Committed it again, let me know what you think ^^